CondFormats/HGCalObjects/interface/HeterogeneousHGCalHEFCellPositionsConditions.h

```diff
@@ -0,0 +1,76 @@
+#ifndef CondFormats_HGCalObjects_HeterogeneousHGCalHEFConditions_h
+#define CondFormats_HGCalObjects_HeterogeneousHGCalHEFConditions_h
+
+#include <numeric>  //accumulate
+#include <typeinfo>
+
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "HeterogeneousCore/CUDACore/interface/ESProduct.h"
+#include "HeterogeneousCore/CUDAUtilities/interface/cudaCheck.h"
+#include "CUDADataFormats/HGCal/interface/HGCConditions.h"
+#include "Geometry/HGCalCommonData/interface/HGCalDDDConstants.h"
+#include "Geometry/HGCalCommonData/interface/HGCalParameters.h"
+#include "RecoLocalCalo/HGCalRecProducers/plugins/KernelManagerHGCalCellPositions.h"
```

Comment:
Hi @b-fontana @cms-sw/reconstruction-l2 @cms-sw/alca-l2 -including stuff from a plugin directory is not allowed in cmssw. (the change also seems contrary to the title of this PR..)

Reply:
it looks it could simply be dropped ?
at least, I don't see `KernelManagerHGCalCellPositions` used anywhere in this file.